### PR TITLE
Send tcp to local servers

### PR DESCRIPTION
**Pull request by john7doe**

### CODE REVIEW

File: ruby-gem/test-server/instrumentation-backend/src/sh/calaba/instrumentationbackend/actions/ipc/SendTcp.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ * Created by john7doe on 22/01/15.
+ */
+
+public class SendTcp implements Action {
+    private static Map<Integer, Socket> connections = new HashMap<Integer, Socket>();
+
+    @java.lang.Override
+    public Result execute(String... args) {
+        if(args.length < 2 || args.length > 3) {
+            return Result.failedResult("Wrong number of arguments");
```
Comment (TobiasRoikjer) [inline ruby-gem/test-server/instrumentation-backend/src/sh/calaba/instrumentationbackend/actions/ipc/SendTcp.java]: I think we should give a better error. Like "Wrong number of arguments (x for 2,3), expected port, command, [keepConnectionAlive]

File: ruby-gem/test-server/instrumentation-backend/src/sh/calaba/instrumentationbackend/actions/ipc/SendTcp.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            } else {
+                socket = connections.get(port);
+            }
+
+            PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
+            BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            out.println(command);
+
+            if(!keepConnectionAlive) {
+                in.close();
```
Comment (TobiasRoikjer) [inline ruby-gem/test-server/instrumentation-backend/src/sh/calaba/instrumentationbackend/actions/ipc/SendTcp.java]: Should we ensure in a finally after the catch, that all open connections will be closed?

**Comment by TobiasRoikjer** [comment]
Overall looks good to me, I have made two comments.

**Comment by john7doe** [comment]
@TobiasRoikjer please merge and make new pre-release. Needed bu UITest

**Pull request merged**